OutOfSchool/OutOfSchool.Common/Config/IMySqlConnectionOptions.cs

```diff
@@ -0,0 +1,19 @@
+namespace OutOfSchool.Common.Config
+{
+    public interface IMySqlConnectionOptions
+    {
+        public bool UseOverride { get; set; }
+
+        public string Server { get; set; }
+
+        public uint Port { get; set; }
```

Comment:
Wow `uint`! First time see this type really used in C# project :)

Reply:
it is used in the Pomelo `MySqlConnectionBuilder` class :(

Reply:
Well, then all credits go to Pomelo author.